packages/plugin-conversation/src/encrypter.js

```diff
@@ -9,7 +9,7 @@ import {isArray, isFunction, isString} from 'lodash';
 import S from 'string';
 
 const encryptableActivities = [
-  `add`,
+  `assign`,
```

Comment:
maybe make a note that assign is a team verb and open a leaky abstract issue about letting plugins add encryptable verbs